spec/system/authentication/user_logs_in_with_email_spec.rb

```diff
@@ -7,6 +7,7 @@
   before do
     allow(Settings::Authentication).to receive(:allow_email_password_registration).and_return(true)
     allow(Settings::Authentication).to receive(:allow_email_password_login).and_return(true)
+    allow(SiteConfig).to receive(:smtp_enabled?).and_return(true)
```

Comment:
it looks like the merge conflict is with the ProfileImageUploader allow also added to the before block

Reply:
Yep, I resolved this. Thank you!